spec/controllers/works_controller_spec.rb

```diff
@@ -327,8 +327,8 @@
 
       let(:uploaded_files) do
         {
-          "0" => uploaded_file1,
-          "2" => uploaded_file2
+          work.pre_curation_uploads[0].key => uploaded_file1,
+          work.pre_curation_uploads[2].key => uploaded_file2
```

Comment:
It surprises me that the numbers aren't in sync... Maybe a comment to explain why?

Reply:
@mccalluc as the test did previously, it is replacing the first and third items. https://github.com/pulibrary/pdc_describe/blob/main/spec/controllers/works_controller_spec.rb#L291 I believe the context explains why not all...